### ISSUE TITLE
A question about Regression-based method

### ISSUE THREAD

**Issue by 674106399** [Jeff-sjtu/res-loglikelihood-regression#3]
Hi Jeff,
Sorry to bother you.

I notice that you classify the Integral Pose method as Heatmap-based method and you use FC to regress the joint coords directly.
I understand that maybe you think the soft-argmax based method is not purely regression-based, but according to my experience, the soft-argmax based method(eg. DSNT, Integral pose) can achieve better performance than fc-based one. 
So I wonder if you have tried ablation experiment like using RLE on Integral Pose Regression?
Looking forward to your reply~
Best,
Tau

**Comment by Jeff-sjtu** [comment]
Hi Tau. Actually, we did conduct experiments on the soft-argmax based method. We use ResNet-50 and using `sigmoid` to predict the normalized heatmap. The original model obtains 65.7 mAP on the COCO validation set, and the model trained with RLE obtain 70.7 mAP. This is slightly better than the direct regression method (70.5 mAP without pretrained). Although using soft-argmax is better, but we think the improvement is marginal and it brings more computation.

**Comment by wyx1997** [comment]
@Tau-J @Jeff-sjtu 
Hi, I wonder that how to combine soft-argmax and RLE together. I mean using soft-argmax can get 2-d coordinate, is this coordinate equals μ, and how can I get the σ?
Looking forward to your reply.
